### PR TITLE
Sticker changes and new functionality

### PR DESCRIPTION
**Pull request by andezdev**
Added new functions and change some interfaces. All this is related with last update of Telegram Bot APi.

Here are the links for the doc related to the interfaces and methods added:

- [Sticker](https://core.telegram.org/bots/api#sticker)
- [StickerSet](https://core.telegram.org/bots/api#stickerset)
- [getStickerSet](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+getStickerSet)
- [getCustomEmojiStickers](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+getCustomEmojiStickers)
- [uploadStickerFile](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+uploadStickerFile)
- [createNewStickerSet](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+createNewStickerSet)
- [addStickerToSet](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+addStickerToSet)
- [setStickerPositionInSet](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+setStickerPositionInSet)
- [deleteStickerFromSet](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+deleteStickerFromSet)
- [setStickerSetThumb](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+setStickerSetThumb)

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -1386,6 +1403,61 @@ declare class TelegramBot extends EventEmitter {
...
+        options?: {},
+        fileOptions?: TelegramBot.FileOptions,
+    ): Promise<TelegramBot.File>;
+
+    createNewStickerSet(
+        userId: string,
+        name: string,
+        title: string,
+        emojis: string,
+        pngSticker?: string | Stream | Buffer,
```
Comment (XC-Zhang) [inline types/node-telegram-bot-api/index.d.ts:1428]: Is order of parameters `pngSticker` and `title` wrong? According to [doc](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#telegrambotcreatenewstickersetuserid-name-title-pngsticker-emojis-options-fileoptions--promise) and [implementation](https://github.com/yagop/node-telegram-bot-api/blob/ccdd14670b10548eda32397cff3f3ed678991f0a/src/telegram.js#L2221), the parameter `pngSticker` is before `emojis`.

Comment (XC-Zhang) [inline types/node-telegram-bot-api/index.d.ts:1428]: Although telegram do support using one of png_sticker, tgs_sticker or webm_sticker, the [implementation](https://github.com/yagop/node-telegram-bot-api/blob/ccdd14670b10548eda32397cff3f3ed678991f0a/src/telegram.js#L2231) only support png_sticker. Could you consider making the parameter `pngSticker` required?

File: types/node-telegram-bot-api/node-telegram-bot-api-tests.ts [hunk 2]
```diff
@@ -350,3 +350,11 @@ MyTelegramBot.setMyDefaultAdministratorRights({
 });
 MyTelegramBot.getMyDefaultAdministratorRights({});
 MyTelegramBot.answerWebAppQuery('query_id', res);
+MyTelegramBot.getStickerSet('custom-set-name');
+MyTelegramBot.getCustomEmojiStickers(['123', '986']);
+MyTelegramBot.uploadStickerFile('user_id', 'my_png_sticker_file');
+MyTelegramBot.createNewStickerSet('user_id', 'short_name', 'my sticker set name', 'hello');
```
Comment (XC-Zhang) [inline types/node-telegram-bot-api/node-telegram-bot-api-tests.ts:356]: Tests would be edited if the definition were changed.

**Review by XC-Zhang — CHANGES_REQUESTED** [review]
See 3 comments. Thanks!

**Comment by toniop99** [comment]
> See 3 comments. Thanks!

Yes, you're absolutely right, I'm going to change it

**Review by XieJiSS — APPROVED** [review]
The latest revision LGTM

**Comment by toniop99** [comment]
Ready to merge

**Pull request merged**